app/src/main/java/com/sunasterisk/moviedb_51/data/source/MovieDataSource.kt

```diff
@@ -1,14 +1,21 @@
 package com.sunasterisk.moviedb_51.data.source
 
+import com.sunasterisk.moviedb_51.data.model.MovieRecent
 import com.sunasterisk.moviedb_51.data.source.remote.response.GenresResponse
 import com.sunasterisk.moviedb_51.data.source.remote.response.MoviesResponse
+import io.reactivex.Flowable
+import io.reactivex.Maybe
 import io.reactivex.Observable
 
 interface MovieDataSource {
     /**
      * Local
      */
     interface Local {
+        fun addMovieRecentLocal(movieRecent: MovieRecent)
+        fun deleteAllMoviesRecent()
+        fun getAllMoviesRecent(): Flowable<List<MovieRecent>>
+        fun findMovieRecent(movieId: Int): Maybe<Int>
```

Comment:
```suggestion
        fun searchMovieLocal(movieId: Int): Maybe<Int>
```
Search thì phải trả về list chứ sao lại trả về int nhỉ?

Reply:
"SELECT count(*) FROM tbl_movie where id = :movieId"
em get count theo id của movie để check sự tồn tại movie đó ( trả về 0 hoặc 1)